benchmarks/deep_learning.py

```diff
@@ -705,7 +705,7 @@ def evaluate_pre_trained_cnn_model(dataset: str, cnn_class: type, config: dict,
         metric_label_prefix="Torch fp32",
     )
 
-    circuit_bitwidth = fhe_module.fhe_circuit.graph.maximum_integer_bit_width()
+    circuit_bitwidth = fhe_module.fhe_circuit.graph.maximum_integer_bit_width(is_encrypted_filter=True)
```

Comment:
What is going on here? The default has changed?

Reply:
Yes the behavior changed, if we want to keep the behavior that we expect this is what we need to do.